letsencrypt/tests/reverter_test.py

```diff
@@ -405,30 +442,35 @@ def setup_test_files():
 
 def get_save_notes(dire):
     """Read save notes"""
-    return read_in(os.path.join(dire, 'CHANGES_SINCE'))
+    return read_in(os.path.join(dire, "CHANGES_SINCE"))
 
 
 def get_filepaths(dire):
     """Get Filepaths"""
-    return read_in(os.path.join(dire, 'FILEPATHS'))
+    return read_in(os.path.join(dire, "FILEPATHS"))
 
 
 def get_new_files(dire):
     """Get new files."""
-    return read_in(os.path.join(dire, 'NEW_FILES')).splitlines()
+    return read_in(os.path.join(dire, "NEW_FILES")).splitlines()
+
+
+def get_undo_commands(dire):
+    """Get new files."""
+    return csv.reader(open(os.path.join(dire, "COMMANDS")))
```

Comment:
yeah... all of these functions do.  I was just tryng to keep it simple because it was a function only used in testing.  Will change though...